### ISSUE TITLE
When generating docs (maximum recursion happens)

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/js-sdk#915]
related to  #367 

### Description

when running `poetry run make docs`

pdoc complains with maximum recursion error reached

```
ImportError: Error importing 'jumpscale.sals.marketplace': maximum recursion depth exceeded
```

### Version information

* OS: e.g. Ubuntu 20.04
* project commit/version: 7454062154eab1629b3a2aeb2e8aede7ce05724d

### Installation method

poetry installation

### Steps to reproduce

run `poetry run make docs`

### Traceback/Logs/Alerts

```
Traceback (most recent call last):
> File "/home/xmonader/wspace/js/js-sdk/.venv/bin/pdoc3", line 11, in <module>
    load_entry_point('pdoc3==0.6.4', 'console_scripts', 'pdoc3')()
    └ <function load_entry_point at 0x7f50d25b5820>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/cli.py", line 448, in main
    modules = [pdoc.Module(module, docfilter=docfilter)
               │    │              │         └ None
               │    │              └ None
               │    └ <class 'pdoc.Module'>
               └ <module 'pdoc' from '/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/__init__.py'>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/cli.py", line 448, in <listcomp>
    modules = [pdoc.Module(module, docfilter=docfilter)
               │    │      │       │         └ None
               │    │      │       └ None
               │    │      └ 'jumpscale'
               │    └ <class 'pdoc.Module'>
               └ <module 'pdoc' from '/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/__init__.py'>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/__init__.py", line 602, in __init__
    self.doc[root] = m = Module(import_module(fullname),
    │    │   │       │   │      │             └ 'jumpscale.sals'
    │    │   │       │   │      └ <function import_module at 0x7f50d1f12dc0>
    │    │   │       │   └ <class 'pdoc.Module'>
    │    │   │       └ <Module 'jumpscale.loader'>
    │    │   └ 'sals'
    │    └ <member 'doc' of 'Module' objects>
    └ <Module 'jumpscale'>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/__init__.py", line 602, in __init__
    self.doc[root] = m = Module(import_module(fullname),
    │    │   │       │   │      │             └ 'jumpscale.sals.marketplace'
    │    │   │       │   │      └ <function import_module at 0x7f50d1f12dc0>
    │    │   │       │   └ <class 'pdoc.Module'>
    │    │   │       └ <Module 'jumpscale.sals.chatflows'>
    │    │   └ 'marketplace'
    │    └ <member 'doc' of 'Module' objects>
    └ <Module 'jumpscale.sals'>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/pdoc/__init__.py", line 185, in import_module
    raise ImportError('Error importing {!r}: {}'.format(module, e))
                                                        └ 'jumpscale.sals.marketplace'

ImportError: Error importing 'jumpscale.sals.marketplace': maximum recursion depth exceeded
```

**Comment by xmonader** [comment]
fixed in 8ecb5d35e09ef63f3864dbc8b7e7563a505a403d

**Comment by Pishoy** [comment]
verified 

```
 cd /opt/js-sdk/
poetry run make docs
python3 -m http.server --directory ./docs 9000
```

- sdk version
```
root@tfwallet-prod:/opt/js-sdk# git log  -1
commit b4e797edde034886342740b966857951faeb6afb (HEAD -> development, origin/development, origin/HEAD)
Author: xmonader <xmonader@gmail.com>
Date:   Sun Sep 6 15:02:35 2020 +0200

    improve error messages for threebot failure and non-reachable websites (#1007)

```
